connector/connect/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -217,3 +218,23 @@ message Sample {
     int64 seed = 1;
   }
 }
+
+// Relation of type [[Range]] that generates a sequence of integers.
+message Range {
+  // Optional. Default value = 0
+  int32 start = 1;
+  int32 end = 2;
+  // Optional. Default value = 1
+  Step step = 3;
```

Comment:
`start`, `end`, `step` should use `int64` @amaliujia

Reply:
Yes let me follow up. I guess I was looking at python side API somehow thus confused myself on the types.

Reply:
Updating in https://github.com/apache/spark/pull/38460.